designsafe/apps/api/search/searchmanager/published_files.py

```diff
@@ -0,0 +1,82 @@
+"""
+.. module: portal.apps.search.api.managers.private_data_search
```

Comment:
I noticed that this manager plus a couple of the other managers (publications and shared data) have this "module" data wrong.  Looks like it was copied from the private_data manager and then not updated for which manager this is.  Not a big deal, since I don't know what uses this, but something I noticed.

Reply:
Thanks, this should be fixed now.